omod/src/main/webapp/resources/jsonforms/HMIS-ACP-008-ComprehensiveARTPatientTransferOutForm.json

```diff
@@ -0,0 +1,1315 @@
+                  {
+                    "concept": "dc882826-30ab-102d-86b0-7a5022ba4115",
+                    "label": "CETIRIZINE"
+                  },
+                  {
+                    "concept": "b04eaf95-77c9-456a-99fb-f668f58a9386",
+                    "label": "OTHER MEDICATIONS DISPENSED"
+                  },
+                  {
+                    "concept": "315825e8-8ba4-4551-bdd1-aa4e02a36639",
+                    "label": "Iron Given"
+                  },
+                  {
+                    "concept": "8c346216-c444-4528-a174-5139922218ed",
+                    "label": "FOLIC ACID Given"
+                  },
+                  {
+                    "concept": "d12abd7f-c90d-4798-9240-0f2f81977183",
+                    "label": "Ctx given to mother"
+                  }
+                ]
+              },
+              "validators": []
+            },
+            {
+              "label": "Other Medications",
+              "type": "obs",
+              "id": "otherMedications",
+              "readonly": true,
+              "questionOptions": {
+                "rendering": "select",
+                "concept": "b04eaf95-77c9-456a-99fb-f668f58a9386",
+                "calculate": {
+                  "calculateExpression": "resolve(api.getLatestObs(patient.id, 'b04eaf95-77c9-456a-99fb-f668f58a9386'))?.valueCodeableConcept?.coding[0]?.code"
+                },
+                "answers": []
+              },
+              "validators": []
+            },
+            {
+              "label": "Any Drug Allergies",
+              "type": "obs",
+              "id": "anyDrugAllergies",
+              "readonly": true,
+              "questionOptions": {
+                "rendering": "select",
+                "concept": "c674105-30ab-102d-86b0-7a5022ba4115",
+                "calculate": {
+                  "calculateExpression": "resolve(api.getLatestObs(patient.id, 'c674105-30ab-102d-86b0-7a5022ba4115'))?.valueCodeableConcept?.coding[0]?.code"
+                },
+                "answers": []
+              },
+              "validators": []
+            }
+          ]
+        },
+        {
+          "label": "Reason for Transfer",
+          "isExpanded": "true",
+          "questions": [
+            {
+              "label": "Reason for Transfer",
+              "type": "obs",
+              "required": true,
+              "id": "reasonForTransfer",
+              "questionOptions": {
+                "rendering": "textarea",
+                "concept": "f88453e3-1676-4b23-9273-39795ac29524"
+              },
+              "validators": []
+            }
+          ]
+        }
+      ]
+    }
+  ],
+  "processor": "EncounterFormProcessor",
+  "uuid": "37be0b17-c62c-4033-90ef-3cbedb7fe0a0",
+  "referencedForms": [],
+  "encounterType": "e305d98a-d6a2-45ba-ba2a-682b497ce27c",
+  "encounter": "Transfer Out",
+  "allowUnspecifiedAll": true,
+  "description": "Patient Transfer Out Form"
```

Comment:
@Daphne210 kindly move these to the top for purposes of consistency

Reply:
> @Daphne210 kindly move these to the top for purposes of consistency

This has been adjusted @slubwama in the subsequent commit